api/src/main/java/org/pmiops/workbench/config/RetryConfig.java

```diff
@@ -67,6 +67,7 @@ protected void logGivingUp(int retryCount, Throwable t) {
     protected Level getLogLevel(int responseCode) {
       switch (responseCode) {
         case HttpServletResponse.SC_NOT_FOUND:
+        case HttpServletResponse.SC_CONFLICT:
```

Comment:
I think this should probably be warning.

Those conflict errors were actually the symptom of RW-580 - not expected.

Reply:
Done.